lms/services/organization.py

```diff
@@ -316,7 +351,7 @@ def usage_report(
                 email=row.email if row.email else "<STUDENT>",
                 h_userid=row.h_userid,
                 course_name=row.course_name,
-                course_created=row.course_created,
+                course_created=row.course_created.isoformat(),
```

Comment:
This is json serializable, the other usage of this reports is dumping it on the admin pages so this change doesn't require any other adjustments.